src/pynwb/data/nwb.misc.yaml

```diff
@@ -76,11 +74,13 @@ groups:
     - name: resolution
       dtype: float
       value: -1.0
-      doc: Value is -1.0
+      doc: Smallest meaningful difference between values in data. Annotations have
+        no units, so the value is fixed to -1.0.
     - name: unit
       dtype: text
       value: n/a
-      doc: Value is 'n/a'
+      doc: Base unit of measurement for working with the data. Annotations have
+        no units, so the value is fixed to 'n/a'.
 
 - neurodata_type_def: IntervalSeries
```

Comment:
Do we have a purpose for this? I feel like the Intervals table can handle this type of data.

Reply:
I agree. This was in the 1.0.6 spec, so I gather it is a legacy item that can now be deprecated. @oruebel @ajtritt what do you think?

Reply:
@rly yes it got carried along from the beginning. I'm not aware of anyone using it now though.